provider/vsphere/ova_import_manager.go

```diff
@@ -89,7 +96,8 @@ func (m *ovaImportManager) importOva(ecfg *environConfig, instSpec *instanceSpec
 		return nil, errors.New(spec.Error[0].LocalizedMessage)
 	}
 	s := &spec.ImportSpec.(*types.VirtualMachineImportSpec).ConfigSpec
-	s.NumCPUs = int(*instSpec.hwc.CpuCores)
+	numCpus := int32(*instSpec.hwc.CpuCores)
```

Comment:
Doesn't the fact that CpuCores is a pointer imply that it is optional and the pointer may be nil?
A nil check seems like it sis required here, same with Mem, CpuPower etc below.